app/components/inplace-select.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 import InPlace from 'ilios/mixins/inplace';
 
+const { observer } = Ember;
+
 export default Ember.Component.extend(InPlace, {
```

Comment:
We should discuss this more in person. I believe `InPlace` mixin was already included before I started.